sds_data_manager/lambda_code/efs_lambda/lambda_function.py

```diff
@@ -55,12 +55,21 @@ def write_data_to_efs(s3_key: str, s3_bucket: str):
 
     logging.debug("After downloading file: %s", os.listdir(mount_path))
 
-    # Attitude naming convention is this:
+    # TODO: we only want historical attitude kernels delivery
```

Comment:
Do you want to add an `and not filename.contains("pred")` check below?

Or possibly we would want to have 3 separate symlinks for users to choose between rather than having a single symlink to be used? `reconstructed`, `historical`, `predict`.